compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -465,7 +496,22 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
       std::vector<float> scaling_factor(size);
       std::vector<int64_t> zp(size);
 
-      quant_bias_per_channel(circle_const, input_scale, weight_scale, scaling_factor, zp);
+      if (output_type == loco::DataType::U8)
+      {
+        quant_bias_per_channel(circle_const, input_scale, weight_scale, scaling_factor, zp);
+      }
+      else if (output_type == loco::DataType::S16)
+      {
+        // Bias is quantized per-tensor in int16 quantization
```

Comment:
Is it OK to do per-tensor quantization in `granularity == ChannelWise` branch?

Reply:
Yes. That's how we decide the scale of bias in CWQ. (we should use one scale for the bias for some reason)

Reply:
I see, thanks